bigquery/google/cloud/bigquery/external_config.py

```diff
@@ -243,8 +247,8 @@ def type_(self, value):
 
     @property
     def columns(self):
-        """List[google.cloud.bigquery.external_config.BigtableColumn]: Lists of
-        columns that should be exposed as individual fields
+        """List[:class:`BigtableColumn`]: Lists of columns that should be
```

Comment:
We probably want to expand this out to the fully-qualified name. I've had docs failures due to an ambiguous class name when other cloud client libraries have the same class name.

Reply:
FWIW, I find that jamming a super-long return type into what is supposed to be a one-line semantic summary is against the spirit of [PEP 257](https://www.python.org/dev/peps/pep-0257).  I don't know why it isn't part of the docstring body, instead.

Reply:
I'm fine with the short name so long as the docs still build. I'm probably being overly cautious since I was bit by it before.

Reply:
I made an update to make these more specific. `` :class:`~.external_config.BigtableColumn` `` renders the same as `` :class:`BigtableColumn` ``, but is less ambiguous. Not putting the full path also helps avoid the unintuitive line breaks required to pass the lint.